code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -310,7 +310,7 @@
 		if(!arrivals_docked)
 			var/obj/screen/splash/Spl = new(character.client, TRUE)
 			Spl.Fade(TRUE)
-			character.playsound_local(get_turf(character), 'sound/voice/ApproachingTG.ogg', 25)
+			character.playsound_local(get_turf(character), 'sound/voice/welcomeBee.ogg', 50)
```

Comment:
Have you checked the volume level in game? The sound file already sounds pretty quiet.

Reply:
I did, sounds good for me, but you know, volume is fucky wucky...